README.md

```diff
@@ -112,3 +111,32 @@ will need to do this binding so the image gets built when maven is run from the
 
 For a complete list of configuration options run:
 `mvn com.spotify:docker-maven-plugin:<version>:help -Ddetail=true`
+
+### Authenticating with Private Registries
+
+To push to a private Docker image registry that requires authentication, you can put your
+credentials in your Maven's global `settings.xml` file. `docker.registry.server-address` is
+optional and defaults to `https://index.docker.io/v1/`.
+
+    <settings>
+      [...]
+      <profiles>
+        <profile>
+          <id>private-docker-registry</id>
+          <properties>
+            <docker.registry.username>foo</docker.registry.username>
```

Comment:
Username/password info is usually part of a `<servers>/<server>` block in `settings.xml`, not a profile.